File: CP/code/intro/factorial.java
import java.util.Scanner;
import java.math.BigInteger;

class factorial {
    public static void main(String[] args) {
        BigInteger fac = BigInteger.ONE;
        for (int i = 2; i <= 25; i++) {
            fac = fac.multiply(BigInteger.valueOf(i));
        }
        System.out.println(fac);
    }
}

File: CP/code/intro/factorial.cs
using System;
using System.Numerics;

class Factorial 
{
    public static void Main(string[] args)
    {
        var fac = BigInteger.One;
        for (int i = 2; i <= 25; i++) 
        {
            fac = fac * i;
        }
        Console.WriteLine(fac);
    }
}